### PR TITLE
[PIE-2574] Add viewing test ownership docs

### PR DESCRIPTION
**Pull request by jasmine-q**
Screenshot 2024-08-06 at 5 45 59 PM

- Added screenshot of the Test ownership tab in Suite settings

### CODE REVIEW

File: pages/test_analytics/test_ownership.md
Comment (gilesgas) [inline pages/test_analytics/test_ownership.md:110]: As indicated in our [writing style guidelines](https://github.com/buildkite/docs/blob/main/styleguides/writing-style.md#ui-elements), I don't see **Suite settings** indicated verbatim within the screenshot you've included.

If **Suite settings** is not indicated anywhere and you were intending to reference the **Test Ownership** tab/page, then I would modify this line to:

```suggestion
You can view the current test ownership rules for a test suite in your **Test Suite** > **Test Ownership** page.
```

The '>' indicating that one might need to click the **Test Ownership** tab to open its page, once initially landing on the **Test Suite** page (which I'm assuming defaults to the **Settings** tab).

File: images/docs/test_analytics/test_ownership/test-ownership.png
Comment (gilesgas) [inline images/docs/test_analytics/test_ownership/test-ownership.png:1]: Is the TESTOWNERS file meant to use `&#39;` for an apostrophe, or does this file support actual apostrophe characters?

If so, would it be possible to retaking this screenshot with a version of the file that indicates `Buildkite's` instead?

**Review by gilesgas — CHANGES_REQUESTED** [review]
Just a few comments that need to be addressed before approval.

File: pages/test_analytics/test_ownership.md
Comment (jasmine-q) [reply pages/test_analytics/test_ownership.md:110]: One would need to navigate from the **Test Suite** page to the **Settings** page which defaults to a **Settings** tab, and finally click on the **Test Ownership** tab.

Would this accurately be represented by the following?

```suggestion
You can view the current test ownership rules for a test suite in your **Test Suite** > **Settings** > **Test Ownership** page
```

File: images/docs/test_analytics/test_ownership/test-ownership.png
Comment (jasmine-q) [reply images/docs/test_analytics/test_ownership/test-ownership.png:1]: > Or, does the actual TESTOWNERS file contain an actual apostrophe and the Buildkite interface is rendering this as the escape code &#39;?

This is the current case

Thanks for raising, I'll have a look into fixing it!

File: pages/test_analytics/test_ownership.md
Comment (gilesgas) [reply pages/test_analytics/test_ownership.md:110]: Thanks for clarifying this @jasmine-q :-)

In that case, it's better to stick with **Test Suite** > **Test Ownership**, since based on the explanation of '>', **Test Suite** > **Settings** > **Test Ownership** implies that you need to click on the **Settings** tab, which from a UX perspective is not the case.

**Review by gilesgas — APPROVED** [review]
Thanks again @jasmine-q ,

If you have time to update the image (assuming the UI fix gets implemented _before_ this docs PR needs to be merged in, and you can take a screenshot of the fixed UI), then please update the screenshot for this PR.

Otherwise, if it'd be better to merge in this docs PR first, please do so. (I'm approving this PR so that you can do this.)

**Comment by jasmine-q** [comment]
> Thanks again @jasmine-q ,
> 
> If you have time to update the image (assuming the UI fix gets implemented _before_ this docs PR needs to be merged in, and you can take a screenshot of the fixed UI), then please update the screenshot for this PR.
> 
> Otherwise, if it'd be better to merge in this docs PR first, please do so. (I'm approving this PR so that you can do this.)

Thanks @gilesgas! [The fix has been merged](https://github.com/buildkite/buildkite/pull/18248) and the screenshot has been updated to include the fix 😄

**Pull request merged**
